core-blocks/paragraph/index.js

```diff
@@ -501,7 +449,7 @@ export const settings = {
 
 		const textClass = getColorClass( 'color', textColor );
 		const backgroundClass = getColorClass( 'background-color', backgroundColor );
-		const fontSizeClass = fontSize && `is-${ fontSize }-text`;
+		const fontSizeClass = getFontSizeClass( 'font-size', fontSize );
```

Comment:
For what reason are we changing the class syntax now?

- Do we need to update `deprecated` reference to the old syntax?
- There are still references to `is-small-text`, etc. in `core-blocks/paragraph/style.scss`

Reply:
I feel using has-large-font-size is more inline with the classes we use for colors, and I would like to change theme.

Block with the old syntax are automatically migrated as classes differences are ignored during validation.

We are keeping the old classes, to not break the existing paragraphs on the front end. I think the solution here is adding a global deprecation notice and keep the classes for two versions.